docs/prd/aws-browser-auth.md

```diff
@@ -0,0 +1,330 @@
+# PRD: AWS Browser-Based Authentication for Root and IAM Identities
+
+**Status:** Draft
+**Author:** Cloud Posse
+**Created:** 2025-12-17
+**Linear Issue:** [DEV-3829](https://linear.app/cloudposse/issue/DEV-3829)
+
+## Overview
+
+Add support for AWS browser-based authentication using the new `aws login` command introduced in AWS CLI 2.32.0. This enables developers to authenticate to AWS using the same credentials they use for the AWS Management Console, eliminating the need for long-term IAM access keys.
+
+## Problem Statement
+
+Currently, atmos supports AWS authentication via:
+- **IAM Identity Center (SSO)** - `aws/iam-identity-center` provider
+- **SAML** - `aws/saml` provider
+- **Static IAM User Credentials** - `aws/user` identity
+
+However, many AWS users authenticate to the console using:
+- Root user credentials
+- IAM user username/password
+- Federated identity providers (non-SSO)
+
+These users currently have no secure way to obtain programmatic credentials in atmos without creating long-term access keys, which AWS discourages as a security risk.
+
+## Solution
+
+Introduce a new `aws/login` provider that leverages the AWS CLI's `aws login` command to obtain temporary credentials through browser-based OAuth2 authentication with PKCE.
```

Comment:
No, this is supported by the SDK; we should integrate with that, not shell out to the `aws login` command.